src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
Got it, thanks. Having the option suggests that it would possibly be used in some semantically meaningful way... since that's not really the case, it was a bit confusing. I'm not really convinced that the convenience is a net win.